src/styles/blocks/hero.scss

```diff
@@ -0,0 +1,42 @@
+.hero {
+  background: url(../images/hero-bg.jpg) left -100px bottom/cover no-repeat;
+
+  @include mediamin(400px) {
+    background-position: left -120px bottom;
+  }
+  @include mediamin(768px) {
+    background-position: center;
+  }
+
+  &__container {
+    @extend %container;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+
+    min-height: 100vh;
+
+    padding-bottom: 94px;
+
+    @include mediamin(768px) {
+      min-height: 100vh;
```

Comment:
Looks like it doesnt change in `@media`))

Reply:
Really :D